larray/core.py

```diff
@@ -10343,12 +10355,13 @@ def empty_like(array, title='', dtype=None, order='K'):
     return LArray(np.empty_like(array.data, dtype, order), array.axes, title)
 
 
+# No need to add @_check_axes_argument, `axes` argument will be checked in empty
```

Comment:
It will return an error if fill_value is an integer. But I can test if fill_value is not an Axis before to call empty inside full